session/tidb_test.go

```diff
@@ -125,21 +123,6 @@ func (s *testMainSuite) TestRetryOpenStore(c *C) {
 	c.Assert(uint64(elapse), GreaterEqual, uint64(3*time.Second))
 }
 
-func (s *testMainSuite) TestRetryDialPumpClient(c *C) {
```

Comment:
remove this testing？ if all pumps were down, what would tidb happen?

Reply:
it is moved to pumps client, pumps client will retry connect to pump

Reply:
if so, it wouldn't return a response, right? is it reasonable?

Reply:
this test is create grpc connection to pump directly, but this is implement in pump client now, so remove DialPumpClientWithRetry function, and don't need this unit test anymore.

Reply:
no, you should add it later. from tidb perspective,  it should know what would happen if there are no available pumps

Reply:
please add a jira issue for it

Reply:
OK， I will create a lira issue

Reply:
https://internal.pingcap.net/jira/browse/TOOL-518